<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>

  <node pkg="realsense_simple" type="realsense_simple" name="realsense_simple">

    <param name="enable_color_image" value="true"/>
    <param name="enable_color_camera_info" value="true"/>
    <param name="enable_depth_image" value="true"/>
    <param name="enable_pointcloud_color" value="true"/>
    <param name="enable_pointcloud_no_intensity" value="false"/>

    <param name="downsample_rate_factor" value="6"/>

    <param name="enable_median_filter" value="true"/>
    <param name="enable_min_max_filter" value="true"/>
    <param name="min_max_filter_size" value="3"/>
    <param name="min_max_filter_threshold" value="0.3"/>

    <param name="enable_color_filter" value="false"/>
    <param name="min_v" value="80"/>
    <param name="max_h" value="90"/>
    <param name="max_s" value="150"/>
  </node>

</launch>
